frontend/src/metabase/collections/components/PinnedItemOverview/PinnedItemOverview.tsx

```diff
@@ -120,3 +187,42 @@ function PinnedItemOverview({
 }
 
 export default PinnedItemOverview;
+
+function getPinDropStyle({
+  hovered,
+  highlighted,
+  isFrontTarget,
+  isBackTarget,
+}: PinDropTargetRenderOptions): React.CSSProperties | undefined {
+  if (hovered) {
+    return {
+      zIndex: 1,
+      position: "absolute",
+      top: 0,
+      bottom: 0,
+      left: 0,
+      right: 0,
+      borderLeft: isFrontTarget
+        ? `4px solid ${color("brand-light")}`
```

Comment:
```suggestion
        ? `4px solid ${color("brand")}`
```

I think making this `brand` here and on line 209 will make this stand out more, though I appeciate how you tried making this look more like my mockup initially.